internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -121,7 +121,6 @@ func (r *AgentServiceConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Result,
 	}
 
 	msg := "AgentServiceConfig reconcile completed without error."
-	r.Recorder.Event(instance, "Normal", aiv1beta1.ReasonReconcileSucceeded, msg)
```

Comment:
This was the event that was unnecessary/confusing to introduce...and the only one that needed to go away.